### ISSUE TITLE
bug: copy_and_exit doesn't copy text

### ISSUE THREAD

**Issue by ziontee113** [rvaiya/warpd#160]
Hi, I'm currently using the option `copy_and_exit: c y`, in Normal Mode, I select the text and press `c or y`, it exits out of Normal Mode but doesn't copy any text. I'd have to press C-C to copy it.

I installed warpd using the AUR package.

**Comment by rvaiya** [comment]
Are you using X? Does this happen everywhere, or only in certain programs? Do you have xclip installed?

**Comment by ziontee113** [comment]
Yes I do use xclip. And I only tested it in Firefox. Is xclip the problem?

**Comment by rvaiya** [comment]
The copy feature works by emitting `control+insert`, which should copy the selection to the primary clipboard and then doing `xclip -o|xclip -selection CLIPBOARD` to make the primary and clipboard selections consistent. I just tested it and it seems to work fine on firefox. Can you see if the problem can be reproduced using other programs?

**Comment by ziontee113** [comment]
Thank you for the answer. I tried it out in `gedit` and it works. I also turned off Vimium C in my Firefox and it also works. So it turns out it's not a `warpd` problem at all :smile: 
However I'm struggling find a way to make this work with Vimium C. I can physically press C-insert and it will copy the text even with Vimium C. But with Vimium C turned on, warpd can't copy text. 

Is there a way that I can set up so that when I press a key, `warpd` will exit and press that key? Thank you very much!

**Comment by ziontee113** [comment]
Come to think of it, I wonder if a "fall through" feature already exists in `warpd`. If a key is not mapped, it acts like `warpd` isn't there. Or maybe something like after selecting with `warpd`, I press `y` or `C-c` and it exits `warpd`, then press the correct keys.

**Comment by rvaiya** [comment]
>  I can physically press C-insert and it will copy the text even with Vimium C.

I will investigate when I have some time.

> Come to think of it, I wonder if a "fall through" feature already exists in warpd. If a key is not mapped, it acts like warpd isn't there.

This has been proposed before, but is non trivial to implement. There are currently no plans for such a feature.
